lib/dependencies-parser.ts

```diff
@@ -50,33 +50,49 @@ export interface PaketDependencies extends Array<DependencyGroup> {
 }
 
 function parseNuget(line: string): NugetDependency {
+  const commentRegex = new RegExp(`(${COMMENTS[0]}|${COMMENTS[1]}).+`);
```

Comment:
doesnt need to be a capturing group. you can change to `(?:${COMMENTS[0]}|${COMMENTS[1]}).+`